src/main/java/com/conveyal/gtfs/util/GeoJsonUtil.java

```diff
@@ -295,6 +291,42 @@ public static CsvReader getCsvReaderFromGeoJson(
         return new CsvReader(new StringReader(csvContent.toString()));
     }
 
+    /**
+     * Extract the locations from Geo Json.
```

Comment:
Fix the GeoJSON spelling (see https://geojson.org/) and update the other comments too.
```suggestion
     * Extract the locations from GeoJSON.
```